sites/footers/footers-en.md

```diff
@@ -0,0 +1,83 @@
+---
+altLangPage: footers-fr.html
+contextualFooter:
+  title: "[Contextual footer header]"
+  links:
+  - url: http://canada.ca/en
+    text: Contextual link 1
+  - url: http://canada.ca/en
+    text: Contextual link 2
+  - url: http://canada.ca/en
+    text: Contextual link 3
+dateModified: 2023-01-16
+includes:
+  footer: edge
+language: en
+secondlevel: false
+share: true
+title: Footer
+---
+
+<div class="wb-prettify all-pre hide"></div>
+
```

Comment:
From @LPipes4:
> I think it was [because] it's launched for real...but if they prefer to keep that till March and then remove it I'd be ok with that too.

Formerly we thought that the documentation update would have been published around the same date as the launch.  But now that the launch has been push back a bit it could return in the documents.